### ISSUE TITLE
Inadequate color contrast: admin-side error text

### ISSUE THREAD

**Issue by aviannamiller** [omeka/omeka-s#2292]
Error text that appears on admin end is failing contrast requirements at 4.2:1 (#A91919 on #F4B4B4). Text color should be altered to #A31313 or darker to ensure proper contrast here, which is at least 4.5:1 for text at this scale.

**Comment by kimisgold** [comment]
The background of error messages has been adjusted and is ready for a look.

**Comment by aviannamiller** [comment]
The contrast looks good.

I noticed error text as it appears on the Modules page is missing a background (receives style from `.error, .error a`), and I'm not sure if it is meant to have a different style, as well as if there are other kinds of error messaging that are also set apart in this way:

<img width="607" alt="Name of a module, 'Custom Advanced Search', with an error message listed next to it. The error message is written in a red font" src="https://github.com/user-attachments/assets/32ff076e-9edb-42e9-8051-65605b8be2f7" />

**Comment by kimisgold** [comment]
I've applied the error background for some consistency, as I don't remember a particular reason for different treatment here.

**Comment by aviannamiller** [comment]
This is great, thanks!
